### PR TITLE
upgrade Terraform and Quorum Provider

### PR DESCRIPTION
**Pull request by trung**
- Migrate to support terraform 0.14.7
- Source Terraform Provider Quorum from Terraform Registry instead of manual download

### CODE REVIEW

File: networks/_modules/ignite/main.tf [hunk 1]
```diff
@@ -1,3 +1,12 @@
+terraform {
+  required_providers {
+    quorum = {
+      source = "trung/quorum"
```
Comment (ricardolyn) [inline networks/_modules/ignite/main.tf:4]: to remove?

Comment (trung) [reply networks/_modules/ignite/main.tf]: changed to `ConsenSysQuorum/quorum` as now we source the provider from our "temporary" official [Terraform Registry](https://registry.terraform.io/providers/ConsenSysQuorum/quorum/latest)

File: networks/_modules/ignite/main.tf [hunk 2]
```diff
@@ -1,3 +1,12 @@
+terraform {
+  required_providers {
+    quorum = {
+      source = "ConsenSysQuorum/quorum"
```
Comment (ricardolyn) [inline networks/_modules/ignite/main.tf:4]: @trung @nmvalera now can we update to `Consensys/quorum`?

Comment (trung) [reply networks/_modules/ignite/main.tf]: :+1: pushed

**Review by ricardolyn — APPROVED** [review]

**Pull request merged**